src/components/notification/sender.ts

```diff
@@ -202,21 +201,14 @@ export const workplaceNotificationSender = async ({
 }
 
 export const desktopNotificationSender = async ({
-  data,
   body,
   status,
 }: {
-  data: DesktopData
   body: string
   status: string
 }) => {
   await sendDesktop({
-    url: data?.url,
-    body: {
-      url: '-',
-      alert: body,
-      time: new Date().toLocaleString(),
-      status,
-    },
+    title: status === 'INIT' ? 'Monika is running' : 'Monika terminated',
```

Comment:
I only read this by code, but if you're gonna do it like this, how are we supposed to get the notifications for the ups and down?

It seems like when there is an alert, it will display "Monika terminated" instead of the alert

Reply:
This is one of the thing that we need to refactor. This sender file only for monika start and termination, while for alert is in /notification/index.ts file